MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -38,10 +38,19 @@
 static NSString *const kMSPushNotificationCustomDataKey = @"mobile_center";
 
 /**
+ * Key for NSUserNotificationCenter delegate property.
+ */
+static NSString *const kMSUserNotificationCenterDelegateKey = @"delegate";
+
+/**
  * Singleton
  */
 static MSPush *sharedInstance = nil;
 static dispatch_once_t onceToken;
+#if TARGET_OS_OSX
+static id<NSUserNotificationCenterDelegate> userNotificationCenterDelegate;
```

Comment:
Can this one be a property instead? And probably renaming it to make it explicit that it's the original userNotificationCenterDelegate not ours.